_layouts/base.html

```diff
@@ -7,6 +7,7 @@
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <meta name="description" content="{% if page.description %}{{ page.description }}{% else %}{{ site.description }}{% endif %}">
+  <link rel="alternate" type="application/rss+xml" title="Bakery Store RSS" href="/blog.atom">
```

Comment:
Bakery store ;) ?

Reply:
Sorry, my bad.

Reply:
No worries. I guess I'd find this on Stack Overflow ;) ?